.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -219,4 +219,28 @@ stages:
           inputs:
             artifactName: 'publishing-info'
             targetPath: 'vhd-publishing-info.json'
-          condition: and(succeeded(), eq(variables.DRY_RUN, 'False'), ne(variables['MODE'], 'sigMode'))
\ No newline at end of file
+          condition: and(succeeded(), eq(variables.DRY_RUN, 'False'), ne(variables['MODE'], 'sigMode'))
+  - stage: build_arm64_vhd
```

Comment:
'dependsOn: []' is still needed here, otherwise the amd64 and arm64 pipeline will run sequentially.